configuration/resize-disk-image.md

```diff
@@ -1,113 +1,89 @@
 ---
 layout: doc
-title: Resize Private Disk Image
+title: Resize Disk Image
 permalink: /doc/resize-disk-image/
 redirect_from:
 - /en/doc/resize-disk-image/
+- /en/doc/resize-root-disk-image/
 - /doc/ResizeDiskImage/
+- /doc/ResizeRootDiskImage/
 - /wiki/ResizeDiskImage/
+- /wiki/ResizeRootDiskImage/
 ---
 
-Resize Private Disk Image
+Resize Disk Image
 -----------------
 
-There are several disk images which can be easily extended, but pay attention to the overall consumed space of your sparse disk images.
-See also additional information and caveats about [resizing the root disk image](/doc/resize-root-disk-image/).
+There are several disk images which can be easily extended, but pay attention to the overall consumed space of your sparse/thin disk images.
+See also [OS Specific Follow-up Instructions](/doc/resize-disk-image/#os-specific-follow-up-instructions) at the end of this page.
 
 
-### Private disk image (R4.0)
+### Template disk image
 
-1048576 MiB is the maximum size which can be assigned to private storage through Qube Manager.
+If you want install a lot of software in your TemplateVM, you may need to increase the amount of disk space your TemplateVM can use. 
+*Make sure changes in the TemplateVM between reboots don't exceed 10G.*
 
-To grow the private disk image of an AppVM beyond this limit, `qvm-volume` can be used:
+1.  Make sure that all the VMs based on this template are shut down (including netvms etc).
+2.  Resize the *root image* using Qubes version specific procedure below.
+   You may instead resize the *private image* with the appropriate commands if your software installs to `/home` for example.
+3.  If any netvm/proxyvm used by this template is based on it, set template's netvm to none.
```

Comment:
Again, this is definitely not needed in 4.0, and probably also not needed in 3.2.

Reply:
I left it in the R3.2 section in case there was some other locking mechanism going on.